nextcloudappstore/core/user/views.py

```diff
@@ -82,3 +86,40 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         context['acc_page'] = 'api-token'
         return context
+
+
+class AppOwnershipTransferView(LoginRequiredMixin, ListView):
+    model = AppOwnershipTransfer
+    template_name = 'user/app-ownership-transfer.html'
+    http_method_names = ['get', 'post']
+
+    def post(self, request, *args, **kwargs):
+        transfer_id = request.POST.get('transfer-id', '')
+        op = request.POST.get('op', '')
```

Comment:
variable should be spelled out, was thinking for a second what the parameter should do